docs/development.rst

```diff
@@ -1,8 +1,18 @@
 Development
 ===========
 
-yamllint provides both a script and a Python module. The latter can be used to
-write your own linting tools:
+``yamllint`` provides both a script and a Python module. The latter can be used
+to write your own linting tools:
+
+Basic example of running the linter from python:
+
+.. code-block:: python
+
+   import yamllint
+
+   yaml_config = yamllint.config.YamlLintConfig("extends: default")
+   for p in yamllint.linter.run(".travis.yml", yaml_config):
```

Comment:
Maybe `example.yaml` would be easier to understand for new users?
And still for consistency, can you please use the `.yaml` extension?